.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -146,12 +146,6 @@ jobs:
         chmod +x /usr/local/bin/updater
         mkdir vulns
 
-    - name: Run updater (single bundle)
```

Comment:
This is here for speed purposes but will be handled via https://github.com/stackrox/stackrox/pull/12610